documentation/modules/operators/proc-configuring-fips-mode-cluster-operator.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-cluster-operator.adoc
+
+[id='proc-configuring-fips-mode-cluster-operator-{context}']
+= Configuring the FIPS mode in Cluster Operator
+
+[role="_abstract"]
+When running Strimzi on FIPS enabled Kubernetes cluster, the OpenJDK used in Strimzi container images will automatically switch to FIPS mode.
```

Comment:
```suggestion
Federal Information Processing Standards (FIPS) are standards for computer security and interoperability.
When running Strimzi on a FIPS-enabled Kubernetes cluster, the OpenJDK used in Strimzi container images will automatically switch to FIPS mode.
```

Reply:
Does the FIPs security level make a difference?

Reply:
What exactly do you mean with FIPS security level? AFAIK on the software level, it is currently just an ON/OFF flag. There are no levels.